### PR TITLE
edit config template

### PR DESCRIPTION
**Pull request by kunmingg**
update config template:
* remove tmp fields
* replace test names

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/3887)

### CODE REVIEW

File: bootstrap/config/kfctl_gcp_basic_auth.yaml [hunk 1]
```diff
@@ -288,64 +279,43 @@ spec:
...
     name: basic-auth-ingress
   email: SET_EMAIL
   enableApplications: true
+  hostname: kf101.endpoints.SET_PROJECT.cloud.goog
+  ipName: kf101-ip
   packageManager: kustomize
   platform: gcp
+  project: SET_PROJECT
+  repos:
+  - name: kubeflow
```
Comment (lluunn) [inline bootstrap/config/kfctl_gcp_basic_auth.yaml]: can you keep the comment:
 # To get manifest at a PR:
   #uri: https://github.com/kubeflow/manifests/archive/pull/235/head.tar.gz

I think it's useful

File: bootstrap/config/kfctl_gcp_basic_auth.yaml [hunk 2]
```diff
@@ -288,64 +279,43 @@ spec:
...
-        value: envoy-ingress
-      - name: issuer
-        value: letsencrypt-prod
       repoRef:
         name: manifests
         path: gcp/basic-auth-ingress
     name: basic-auth-ingress
   email: SET_EMAIL
   enableApplications: true
+  hostname: kf101.endpoints.SET_PROJECT.cloud.goog
```
Comment (lluunn) [inline bootstrap/config/kfctl_gcp_basic_auth.yaml]: This two fields will get override, right?

File: bootstrap/config/kfctl_gcp_basic_auth.yaml [hunk 3]
```diff
@@ -288,64 +279,43 @@ spec:
...
-        name: manifests
-        path: common/ambassador
-    name: ambassador
   - kustomizeConfig:
       repoRef:
         name: manifests
         path: common/basic-auth
     name: basic-auth
   - kustomizeConfig:
-      overlays:
```
Comment (lluunn) [inline bootstrap/config/kfctl_gcp_basic_auth.yaml]: Why remove overlays?

File: bootstrap/config/kfctl_gcp_basic_auth.yaml [hunk 4]
```diff
@@ -257,17 +256,9 @@ spec:
         path: cert-manager/cert-manager
     name: cert-manager
   - kustomizeConfig:
-      overlays:
```
Comment (lluunn) [inline bootstrap/config/kfctl_gcp_basic_auth.yaml]: why remove overlay?

**Comment by kunmingg** [comment]
Let's close this one as the config file will be used on v0.6 branch.
New PR: https://github.com/kubeflow/kubeflow/pull/3892
@lluunn

**Pull request closed without merging**